lib/src/platform_client.dart

```diff
@@ -305,6 +306,24 @@ class PlatformClient {
     return [];
   }
 
+  /// Gets the status of a service request.
+  ///
+  /// If the `status` is `ASSIGNED`, this will also return the `agentFirstName`.
+  ///
+  /// This API is not intended for public consumption and is subject to change.
+  @internal
+  Future<Map<String, String?>> getServiceRequestStatus(int serviceRequestId) async {
+    _verifyIsLoggedIn();
+
+    Map<String, dynamic> response =
+        await _httpGet('/api/user/service', queryParameters: {'id': serviceRequestId.toString()});
+
+    return {
+      'agentFirstName': response['agentName']?.toString().split(' ').first, // Split the first name and last initial.
+      'status': response['serviceStatus'],
+    };
```

Comment:
At the moment, we just return the `agentFirstName` and `status`. In the future, we could potentially return things like the position in queue and the expected wait time. 🤔